src/data/MediaProviderService.js

```diff
@@ -0,0 +1,135 @@
+import ApiService from './ApiService'
+
+/**
+ * Service that calls API to get Media Provider stats
+ * @param {'image'|'audio'} mediaType
+ * @constructor
+ */
+const MediaProviderService = (mediaType) => ({
+  /**
+   * Implements an endpoint to get audio provider statistics.
+   * SSR-called
+   */
+  getProviderStats() {
+    try {
+      // TODO: use the new 'image/stats' endpoint when it is available in API
+      if (mediaType === 'image') {
+        return ApiService.get('', 'sources')
+      }
+      return ApiService.get(mediaType, 'stats')
+    } catch (err) {
+      console.log('Error ', err)
+    }
+  },
+  getProviderInfo(providerName) {
```

Comment:
It seems redundant to classify providers by media type here because the logo for the provider will be the same irrespective of the media type. Classifying by media type will require multi-media providers to be listed multiple times.